src/theme/animations.ts

```diff
@@ -0,0 +1,28 @@
+import { css, keyframes } from 'styled-components/macro'
+
+const transitions = {
+  duration: {
+    slow: '500ms',
+    medium: '250ms',
+    fast: '125ms',
+  },
+  timing: {
+    ease: 'ease',
+    in: 'ease-in',
```

Comment:
Unless we are never going to use another function (like linear), I'd strongly prefer these be named `easeIn`, `easeOut`, `easeInOut`.